nilearn/datasets/atlas.py

```diff
@@ -808,6 +809,79 @@ def fetch_coords_dosenbach_2010(ordered_regions=True):
     return Bunch(**params)
 
 
+def fetch_coords_seitzman_2018(ordered_regions=True):
+    """Load the Seitzman et al. 300 ROIs. These ROIs cover cortical,
+    subcortical and cerebellar regions and are assigned to one of 13
+    networks (Auditory, CinguloOpercular, DefaultMode, DorsalAttention,
+    FrontoParietal, MedialTemporalLobe, ParietoMedial, Reward, Salience,
+    SomatomotorDorsal, SomatomotorLateral, VentralAttention, Visual) and
+    have a regional label (cortexL, cortexR, cerebellum, thalamus, hippocampus,
+    basalGanglia, amygdala, cortexMid).
+
+    .. versionadded:: 0.5.1
+
+    Parameters
+    ----------
+    ordered_regions : bool, optional
+        ROIs from same networks are grouped together and ordered with respect
+        to their locations (anterior to posterior).
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, contains:
+        - "rois": Coordinates of 300 ROIs in MNI space
+        - "radius": Radius of each ROI in mm
+        - "networks": Network names
+        - "regions": Region names
+
+    References
+    ----------
+    Seitzman, B. A., Gratton, C., Marek, S., Raut, R. V., Dosenbach, N. U.,
+    Schlaggar, B. L., et al. (2018). A set of functionally-defined brain
+    regions with improved representation of the subcortex and cerebellum.
+    bioRxiv, 450452. http://doi.org/10.1101/450452
+    """
+    dataset_name = 'seitzman_2018'
+    fdescr = _get_dataset_descr(dataset_name)
+    package_directory = os.path.dirname(os.path.abspath(__file__))
+    roi_file = os.path.join(package_directory, "data",
+                            "seitzman_2018_ROIs_300inVol_MNI_allInfo.txt")
+    anatomical_file = os.path.join(package_directory, "data",
+                                   "seitzman_2018_ROIs_anatomicalLabels.txt")
+
+    rois = np.recfromcsv(roi_file, delimiter=" ", encoding="ascii")
+    rois = recfunctions.rename_fields(rois, {"netname": "network",
+                                             "radiusmm": "radius"})
+
+    # get integer regional labels and convert to text labels with mapping
+    # from header line
+    with open(anatomical_file, 'r') as fi:
+        header = fi.readline()
+    region_mapping = {}
+    for r in header.strip().split(","):
+        i, region = r.split("=")
+        region_mapping[int(i)] = region
+
+    anatomical = np.genfromtxt(anatomical_file, skip_header=1,
```

Comment:
Can we use ```from nilearn._utils.numpy_conversions import csv_to_array``` here ?

Reply:
`csv_to_array` removes the header, which is not ideal for here